src/Directory.Build.props

```diff
@@ -22,12 +22,18 @@
     <PackageIcon>sentry-nuget.png</PackageIcon>
     <PackageProjectUrl>https://sentry.io</PackageProjectUrl>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
 
     <PackageReleaseNotes>Can be found at: https://github.com/getsentry/sentry-dotnet/releases</PackageReleaseNotes>
 
     <!-- Prop required by Microsoft.SourceLink.GitHub -->
+    <!-- Optional: Publish the repository URL in the built .nupkg (in the NuSpec <Repository> element) -->
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
```

Comment:
Is this true by default? I wonder because sourcelink already worked on this repo

Reply:
Another great question. I found [the docs on this](https://github.com/dotnet/sourcelink/blob/master/docs/README.md#publishrepositoryurl): (Emphasis, mine)

> The URL of the repository supplied by the CI server or retrieved from source control manager is stored in PrivateRepositoryUrl variable.
>
>This value is not directly embedded in build outputs to avoid inadvertently publishing links to private repositories. Instead, PublishRepositoryUrl *needs to be set by the project* in order to publish the URL into RepositoryUrl property, which is used e.g. in the nuspec file generated for NuGet package produced by the project.

To me, this suggests that the default is `FALSE` .. but if that's the case ... how does this work?

I've removed it (locally) and generated a new package .. and  .. it lists the url in there!

I .. don't get it :(

I _think_ this value is _different_ to `<RepositoryUrl>https://github.com/getsentry/sentry-dotnet</RepositoryUrl>` which already exists.

So maybe:
- `<RepositoryUrl>` can be anything, which is meta data for someone to 'know' where the repo is if they want to _manually_ look at code
- `<PublishRepositoryUrl>` maybe related to how the symbol packages are known where to exist? 

I'm thinking, based on the docs, I should put it in there .. mainly because this is a PUBLIC repo.

Reply:
Also, [asked a question](https://github.com/dotnet/sourcelink/issues/659) in the sourcelink repo.